public/js/semantic.dropdown.js

```diff
@@ -0,0 +1,4019 @@
+/*!
+ * # Semantic UI 2.3.1 - Dropdown
+ * http://github.com/semantic-org/semantic-ui/
+ *
+ *
+ * Released under the MIT license
+ * http://opensource.org/licenses/MIT
+ *
+ */
+
+/* This version has been modified for Gitea */
```

Comment:
```suggestion
/*
 * Copyright 2019 The Gitea Authors. All rights reserved.
 * This version has been modified by Gitea to improve accessibility.
 */
```

I'm not sure why the first comment starts with `/*!` instead of `/*`, though.

Reply:
Can I leave out the 'All rights reserved.'? Seems a bit weird given it's under the MIT license

Reply:
I think you have to still put it as it's standard boilerplate. I'd probably put the comment before the copyright so as to make it clear that only the modifications are Gitea's but it's ok